packages/components/src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import * as ToastPrimitive from "@radix-ui/react-toast";
+import { Anchor } from "../Anchor";
+import { Box } from "../../primitives/Box";
+import { Text } from "../../primitives/Text";
+import { Icon } from "../Icon";
+import type { ToastVariants } from "./types";
+
+export interface ToastProps
+  extends Omit<React.ComponentProps<typeof ToastPrimitive.Root>, "asChild"> {
+  /** The text used if an Anchor is used in the toast. */
+  anchorText?: string;
+  /** The href used if an Anchor is used in the toast. */
+  anchorHref?: string;
+  /** The text content of the toast. */
+  children: NonNullable<React.ReactNode>;
+  /** The type and style of the toast. */
+  variant: ToastVariants;
+}
+
+/** A succinct message that is displayed temporarily. */
+const Toast = React.forwardRef<HTMLLIElement, ToastProps>(
+  ({ anchorText, anchorHref, children, variant, ...props }, ref) => {
+    return (
+      <Box.li
+        as={ToastPrimitive.Root}
+        backgroundColor={
+          variant === "error"
+            ? "colorBackgroundError"
+            : "colorBackgroundSuccess"
+        }
+        borderColor={
+          variant === "error" ? "colorBorderError" : "colorBorderSuccess"
+        }
+        borderRadius="borderRadius20"
+        borderStyle="borderStyleSolid"
+        borderWidth="borderWidth10"
+        boxShadow="shadow"
+        display="grid"
+        gap="space40"
+        gridTemplateColumns="20px 1fr 20px"
+        padding="space40"
+        ref={ref}
+        {...props}
+      >
+        <Icon
+          color={variant === "error" ? "colorIconError" : "colorIconSuccess"}
+          decorative
+          icon={variant === "error" ? "XCircleIcon" : "CheckCircleIcon"}
+          size="sizeIcon30"
+        />
+
+        <Box.span>
+          <ToastPrimitive.Description asChild>
+            <Text.span
+              color={
+                variant === "error" ? "colorTextError" : "colorTextSuccess"
+              }
+              fontFamily="fontFamilyModerat"
+              fontSize="fontSize30"
+              fontWeight="fontWeightMedium"
+              lineHeight="lineHeight30"
+            >
+              {children}
+            </Text.span>
+          </ToastPrimitive.Description>
+          {anchorHref && anchorText && (
+            <>
+              {" "}
+              <ToastPrimitive.Action altText={anchorText} asChild>
+                <Anchor href={anchorHref}>{anchorText}</Anchor>
```

Comment:
Are we sure this will alway be a link and might not need to call a function instead? Maybe a success toast with a undo action for example?

Reply:
I made the cta prop render anything, but for now we can only use Anchors because Button needs to be reworked so it would fit inline with text. I added a comment to the story, and I'll add a ticket to handle the Button changes.